README.md

```diff
@@ -0,0 +1,210 @@
+# Global CO2 from Cement Production Dataset
+
+This repository downloads the Andrew dataset on global CO2 emissions from cement production from [Zenodo](https://zenodo.org/records/831454).
+The dataset is converted to the PRIMAP2 format and provided in the csv based interchange format and the netCDF based native primap2 format. Several version of the dataset are available.
+
+## Description
+
+This repository downloads data on global CO2 emissions from cement production from [Zenodo](https://zenodo.org/records/831454).
+The downloaded dataset can then be converted into CSV (.csv file extension) or NetCDF (.nc file extension) format. Converted data are available for the following versions:
+
+| v231016 |[Zenodo](https://zenodo.org/records/10008931) |
+| v230913 |[Zenodo](https://zenodo.org/records/8339353) |
+| v230428 |[Zenodo](https://zenodo.org/records/7875557) |
+| v220915 |[Zenodo](https://zenodo.org/records/7081360) |
+| v220516 |[Zenodo](https://zenodo.org/records/6553090) |
+The data management tool [DataLad](http://docs.datalad.org/en/stable/) is used to version control the data sets.
+Commands to run the scripts are executed via the pydoit package.
+
+## DataLad datasets and how to use them  
+  
+This repository is a [DataLad](https://www.datalad.org/) dataset. It provides  
+fine-grained data access down to the level of individual files, and allows for  
+tracking future updates. In order to use this repository for data retrieval,  
+[DataLad](https://www.datalad.org/) is required. It is a free and open source  
+command line tool, available for all major operating systems, and builds up on  
+Git and [git-annex](https://git-annex.branchable.com/) to allow sharing,  
+synchronizing, and version controlling collections of large files.  
+
+## Installation
+
+- Install datalad according to the [DataLad handbook](https://handbook.datalad.org/en/latest/intro/installation.html). It is recommended to install globally. 
+- Install [Python](https://www.python.org)
+- [pydoit](https://pydoit.org/install.html)
+
+Note that for [simply downloading the dataset,](#1-easy-access) Python and pydoit are not required.
+
+## Getting Started
+
+### Clone the repository
+
+A DataLad dataset can be `cloned` by running  
+```
+datalad clone
+```
+Do not use **git clone** to download the repository! This way DataLad will not have the necessary
```

Comment:
```suggestion

Do not use git clone to download the repository! If you use plain git clone, DataLad will not have the necessary
```

New line after code block is convention.

Language changes make things a bit clearer I think